src/models/OtrStore.js

```diff
@@ -24,6 +24,9 @@ function OtrStore(name, client) {
 OtrStore.prototype.initChan = function(args) {
 	var network_id = args.network;
 	var chan = args.chan;
+	var otrStore = this;
+	var client = this.client;
+
 	if (chan.type === Chan.Type.QUERY) {
 		var network = _.find(this.client.networks, {id: network_id});
```

Comment:
That's not systematic and depends on lines size, see https://github.com/astorije/shout/blob/master/src/client.js#L106 or https://github.com/astorije/shout/blob/master/src/client.js#L291 https://github.com/astorije/shout/blob/master/src/plugins/inputs/msg.js#L23